src/components/bs5/video/video.stories.js

```diff
@@ -0,0 +1,174 @@
+// video.stories.js
+import { Video } from './Video.js';
+import defaultdata from './video.data.json';
+
+// include accordion for transcript
+import { Accordion } from "../accordion/Accordion.js";
+
+export default {
+  tags: ["autodocs"],
+  title: "Components/Video",
+  args: defaultdata.youtube,
+  render: (args) => {
+    const transcript = new Accordion({
+      groupid: `video-transcript-${args.videoId}`,
+      accordionItems: [
+        {
+          id: args.videoId,
+          title: "Show transcript",
+          expanded: false,
+          content: args.transcriptContent,
+        },
+      ],
+    }).html;
+    return new Video({ ...args, transcript: transcript }).html;
+  },
+
+  argTypes: {
+    source: {
+      control: {
+        type: "radio",
+      },
+      options: {
+        Youtube: "youtube",
+        Vimeo: "vimeo",
+      },
+    },
+    videoSize: {
+      control: {
+        type: "radio",
+      },
+      options: {
+        "Full width": "col-12",
+        "Two thirds": "col-8",
+        "Half page": "col-6",
+      },
+    },
+  },
+
+  /**
+   * Additional parameters for the story.
+   *
+   * @type {Object}
+   * @property {Object} design - Configuration for the design parameter.
+   * @property {string} design.name - Name of the design parameter.
+   * @property {string} design.type - Type of the design parameter.
+   * @property {string} design.url - URL of the design parameter.
+   */
+  parameters: {
+    // design: {
+    //   name: "QGDS Figma Reference",
+    //   type: "figma",
+    //   url: "https://www.figma.com/file/qKsxl3ogIlBp7dafgxXuCA/QLD-GOV-DDS?type=design&node-id=24025-293663&mode=design&t=uf49yHsvyUfA39Lv-4",
+    // },
+    docs: {
+      controls: {
+        exclude: ["transcript"],
+      },
+    },
+  },
+};
+
+/**
+ * Default Video
+ */
+export const Default = {};
+
+/**
+ * Vimeo Video
+ */
+export const Vimeo = {
+  args: defaultdata.vimeo,
```

Comment:
Can you add
```
 chromatic: { disableSnapshot: true }, //ignoring snapshot since Vimeo is slow return and gives blank window.
 ```

Reply:
Hey @duttonw, this story uses a local image for the thumbnail - `/assets/img/banner-example-3-to-2.jpg`. Should these images be available in the github served Storybook/Chromatic?